### PR TITLE
Adding Europe's Flag

### PR DESCRIPTION
**Pull request by brunocavalcantelima**
We request the addition of the flag of Europe, Here's the example:

bandeira

### CODE REVIEW

**Comment by norkans7** [comment]
@brunocavalcantelima you need to add favico file as well,
Check #1073

**Review by norkans7 — CHANGES_REQUESTED** [review]
favico missing

**Comment by brunocavalcantelima** [comment]
> @brunocavalcantelima you need to add favico file as well, Check #1073



> @brunocavalcantelima you need to add favico file as well, Check #1073

Add favico

File: ureport/settings_common.py [hunk 1]
```diff
@@ -1183,6 +1183,13 @@
         countries_codes=["SWZ"],
         count_link="http://eswatini.ureport.in/count/",
     ),
+    dict(
+        name="Europe",
+        host="//europe-uk.ureport.in/",
+        flag="flag_europe.png",
```
Comment (norkans7) [inline ureport/settings_common.py:1189]: use `flag_europe-uk.png` for consistency with the subdomain

File: ureport/settings_common.py [hunk 2]
```diff
@@ -1183,6 +1183,13 @@
...
+        countries_codes=["EUR"],
```
[the remark this answers is not in the document]
Comment (brunocavalcantelima) [reply ureport/settings_common.py]: I made the suggested updates.

**Review by norkans7 — APPROVED** [review]

**Pull request merged**